rwkv/chat_with_bot.py

```diff
@@ -1,11 +1,13 @@
 # Provides terminal-based chat interface for RWKV model.
 
+import os
 import sys
 import argparse
 import sampling
 import tokenizers
 import rwkv_cpp_model
 import rwkv_cpp_shared_library
+from pathlib import Path
```

Comment:
I would prefer to use `import pathlib ` and `pathlib.Path` below for style consistency